data/json/itemgroups/main.json

```diff
@@ -0,0 +1,227 @@
+[
+  {
+    "type" : "item_group",
+    "id" : "trader_avoid",
```

Comment:
This group is currently specified twice: here and in the old, big itemgroups.json